src/App.js

```diff
@@ -1,13 +1,28 @@
 import { HashRouter, Routes, Route } from "react-router-dom";
+import React, { useState, useEffect } from "react";
+
 import Home from "./pages/Home";
+import About from "./pages/About";
 import NoPage from "./pages/NoPage";
+import { Photos } from "./pages/Photos";
```

Comment:
we should use git lfs for storing the photos - https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage
would reduce size since we have a lot of pngs now. because of all the commits this project is almost a gig (max is 2 gigs) and every commit adds a bunch more. keeping it on LFS and deleting this branch / these commits would make it like 100 megs forever